substreams/crates/tycho-substreams/src/testing/assets.rs

```diff
@@ -0,0 +1,10 @@
+pub fn read_block<B: prost::Message + Default>(filename: &str) -> B {
```

Comment:
Can you document this please? I think it'll be very helpful for the integration process